Resources/public/js/ezrecommendation.js

```diff
@@ -0,0 +1,77 @@
+// register HandleBars helpers when library is loaded
+
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
doing that on `DOMContentLoaded` is a bit weird since you don't the page to be loaded at all, you just need to execute that after the loading of handlebars.js (how is it done by the way ?)